src/builtin-templates/hosted-payment-card-form.js

```diff
@@ -1,29 +1,42 @@
 const BASE_URL = 'https://vault.automationcloud.net/forms/index.html';
 const DEFAULT_OPTIONS = {
-    fields: 'pan,name,expiry-select,cvv',
-    brands: 'visa,mastercard,amex',
-    validateOnInput: 'on',
-    css: 'https://ubio-application-bundle-dummy-server.glitch.me/style.css'
+    fields: ['pan', 'name', 'expiry-select', 'cvv'],
+    brands: ['visa', 'mastercard', 'amex']
 };
 
+function formatFields(fields) {
+    const formatted = [];
+
+    for (const element of fields) {
+        formatted.push(typeof element === 'string' ? element : `${element.field}_${element.name}`);
+    }
+
+    return formatted.join(',');
+}
+
+function formatBrands(brands) {
+    return brands.join(',');
+}
+
 export default (otp, formOptions = {}, formStyles = {}) => {
-    const options = {
-        ...DEFAULT_OPTIONS,
-        ...formOptions
-    };
+    if (!formOptions.css) {
+        throw new Error('css is a required field option.');
+    }
 
-    const optionsString = Object.keys(options)
-        .filter(key => options[key])
-        .map(key => `&${key}=${options[key]}`)
-        .join('');
+    const search = new URLSearchParams([
+        ['otp', otp],
+        ['fields', formatFields(formOptions.fields || DEFAULT_OPTIONS.fields)],
+        ['brands', formatBrands(formOptions.brands || DEFAULT_OPTIONS.brands)],
+        ['validateOnInput', formOptions.validateOnInput === false ? 'off' : 'on'],
```

Comment:
I guess it's the other way around in our official docs: 'on' is an only way to enable (and afaik it's a default), and everything else that is not `'on'` defaults to `'off'`

Reply:
This module was originally setting the default to `true`, effectively contradicting the README. This update preserves the existing behaviour of the code (which I can update the README to reflect).